stdlib/websub/src/main/ballerina/websub/Module.md

```diff
@@ -344,3 +390,92 @@ dispatching should happen to a resource named `onIssueOpened`.
 The first parameter of this resource will be the generic `websub:Notification` record and the second parameter will 
 be a custom `IssueOpenedEvent` record, mapping the JSON payload received when an issue is created.  
  
+#### Sample Specific Subscriber Service
+
+In order to introduce a specific subscriber service, a new Ballerina `listener` needs to be introduced, wrapping the 
```

Comment:
```suggestion
In order to introduce a specific subscriber service, a new Ballerina `listener` needs to be introduced by wrapping the 
```

Reply:
Wrapping the `websub:Listener` is not mandatory to introduce a Ballerina `listener`, IMO adding "by" kinda makes it sound mandatory. WDYT?